ci/scripts/shared_utilities.sh

```diff
@@ -31,7 +31,7 @@ find-here-test-reports() {
 }
 
 ## Parsing functions for the Concourse Semver resource.
-## These functions expect one input in the form of the resource file, e.g., "1.9.0-SNAPSHOT.325"
+## These functions expect one input in the form of the resource file, e.g., "1.9.0-build.325"
```

Comment:
Can we use 1.14.0 here instead of an old version?

Reply:
Yes. I'll change it during the rebase (rebasing due to conflict).